README.md

```
@@ -33,4 +33,4 @@ For general support or usage questions, use the [Auth0 Community](https://commun
 
 <p align="center">Auth0 is an easy-to-implement, adaptable authentication and authorization platform. To learn more check out <a href="https://auth0.com/why-auth0">Why Auth0?</a></p>
 
-<p align="center">This project is licensed under the MIT license. See the <a href="./LICENSE"> LICENSE</a> file for more info.</p>
\ No newline at end of file
+<p align="center">This project is licensed under the Apache 2.0 license. See the <a href="./LICENSE">LICENSE</a> file for more info.</p>
```

markdownlint-cli2 (0.23.2)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)